### PR TITLE
feat(icons): add icons for dataStreams components

### PR DESCRIPTION
**Pull request by maximenicoul**
components concerned: Window, Replicate, DBinput

**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
No such icons available

**What is the new behavior?**
Window, Replicate and DBInput icons available

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

File: packages/icons/src/svg/placeholder.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50">
```
Comment (acateland) [inline packages/icons/src/svg/placeholder.svg]: this seam to be overly complex for a simple icon, maybe path should be optimized a bit ?

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://296.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://296.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://296.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://296.talend.surge.sh)

**Pull request merged**